### DISCUSSION TITLE
请问如何加入循环事件

### DISCUSSION THREAD

**Discussion by applelrq** [GraiaProject/Application#121] [Q&A]
比如我想除了能监听QQ消息之外， 还想加入一个while true循环能监听直播是否开播。
我的代码如下
@bcc.receiver("GroupMessage")
async def  #group_message_handler(
    message: MessageChain,
    app: GraiaMiraiApplication,
    group: Group, member: Member,
):

#监听QQ群消息
app.launch_blocking()

请问在哪里可以加入whie true来监听直播是否开播

**Comment by GreyElaina** [comment]
建议 graia scheduler

**Reply by gogolrq** [reply]
多谢已经解决。
